src/libraries/System.Collections.Immutable/ref/System.Collections.Immutable.cs

```diff
@@ -268,8 +268,7 @@ public static partial class ImmutableDictionary
         public static System.Collections.Immutable.ImmutableDictionary<TKey, TValue> Create<TKey, TValue>() where TKey : notnull { throw null; }
```

Comment:
Above on row 125-127 `public System.Collections.Immutable.ImmutableArray<TOther> As<TOther>() where TOther : class?` we make generic type `TOther` derived from `class?` which might make the returning `ImmutableArray<TOther>` return nullable elements, but not sure if deriving from nullable type makes the type always be nullable:
`public System.Collections.Immutable.ImmutableArray<TOther> As<TOther>() where TOther : class?`
 Related to comment: https://github.com/dotnet/runtime/pull/40084/files#r463896011

Reply:
@buyaa-n, can you please address that additional feedback on immutable collections in a separate issue / PR once this one goes in?  Thanks.

Reply:
Sure, created an [issue ](https://github.com/dotnet/runtime/issues/40481)